lametro/templatetags/lametro_extras.py

```diff
@@ -327,3 +330,10 @@ def get_events_with_manual_broadcasts():
     broadcasts = EventBroadcast.objects.filter(is_manually_live=True)
     events = [b.event for b in broadcasts]
     return events
+
+
+@register.filter
```

Comment:
To avoid repeated database calls for the latest action, I annotated the event's board reports with its latest action's description. Then, I use this template filter to parse the bill status.